modules/ROOT/pages/fix-instance-conformance-issues.adoc

```diff
@@ -0,0 +1,63 @@
+= Finding and Fixing Conformance Issues in API Instances
+
+After API Governance identifies conformance issues in your API instances, you can view the governance validation report details to get the information you need to fix the issues. After you identify what needs to be fixed, go to the instance in API Manager and fix the issue. If the issue is an instance-related issue in the specification, use your API editor, such as API Designer, to fix the specification.
+
+// Screenshot of a Conformance Report page with callouts 
+
+include::_partials/api-gov-apim-governance-report.adoc[]
+
+== View Conformance Issue Details in the Governance Report
+
+To view instance conformance issues in the governance validation report:
+
+. Navigate to the report.
+* In API Governance:
++
+.. In the API Governance console, select the *Governed APIs* tab.
+.. If the *Instances* column has a *Nonconformant* badge, click *View Report* for information about the conformance issue.
+* In API Manager:
+.. Select the *Conformance Report* page for the API that is marked as nonconformant.
+. View nonconformance details for each ruleset:
++
+* Click the icon for the nonconformance severity to see details for issues for that severity level. 
+* Click *View Details* to see the full breakdown of conformance issues.
+
+== View Rule Details in the Rulesets
+
+If the nonconformance messages in the governance validation report don't have enough detail to help you fix the issues, you can view more detailed information in the rulesets. 
+
+To view rules details in rulesets:
```

Comment:
```suggestion
```

Reply:
Are we prohibited from using this style? I saw in the standards that it's optional. I like the consistency of it for scanning for tasks.

Reply:
I realize that the trade-off is some duplication in this case.